pkg/model/manifest.go

```diff
@@ -721,6 +711,33 @@ func (b *ManifestBuild) validate() error {
 	return nil
 }
 
+func (s *Secret) validate() error {
+	if s.LocalPath == "" || s.RemotePath == "" {
+		return fmt.Errorf("secrets must follow the syntax 'LOCAL_PATH:REMOTE_PATH:MODE'")
+	}
+
+	if err := checkFileAndNotDirectory(s.LocalPath, afero.NewOsFs()); err != nil {
+		return err
+	}
+
+	if !strings.HasPrefix(s.RemotePath, "/") {
+		return fmt.Errorf("secret remote path '%s' must be an absolute path", s.RemotePath)
+	}
+
+	return nil
+}
+
+func checkFileAndNotDirectory(path string, fs afero.Fs) error {
```

Comment:
I think this code is repeated in `CheckIfRegularFile`. Should we create a new package that takes care of the filesystem? I think that could reduce the `cmd/utils`

Reply:
I've seen that this was moved from the serializer. Up to you if we change it now or when we get to that part

Reply:
I will open a follow up PR for this, because there are a few changes that I want to make there.